crypto/fipsmodule/evp/p_ec.c

```diff
@@ -155,6 +155,7 @@ static int pkey_ec_derive(EVP_PKEY_CTX *ctx, uint8_t *key,
   eckey = ctx->pkey->pkey.ec;
 
   if (!key) {
+    // This only outputs the expected |keylen| and does no actual crypto.
```

Comment:
l. 170: may be add a comment before the call to ECDH_compute_shared_secret stating that it's an internal function that does not set the service indicator.

Reply:
Done.